packages/app/src/components/ColorSchemeSwitcher.module.css

```
@@ -0,0 +1,45 @@
+}
+
+.option:hover {
+  background: #eee;
+}
+
+.active {
+  background: #ddd;
+  border-color: #999;
+}
+
+:global([data-color-scheme="dark"]) .switcher {
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)